src/main/java/codesquad/springcafe/repository/ArticleRepository.java

```diff
@@ -2,26 +2,13 @@
 
 import codesquad.springcafe.model.Article;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 
-public class ArticleRepository {
+public interface ArticleRepository {
+    void addArticle(Article article);
```

Comment:
정답은 없지만 JPA CrudRepository 에서는 요런식으로 인터페이스를 제공하고 있으니 참고해주세요
```suggestion
    Article save(Article article);
```

Reply:
@JinseongHwang 확인해보겠습니다 감사합니다👍